pkgs/by-name/me/mediasynclite/package.nix

```diff
@@ -0,0 +1,52 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, gtk3
+, glib
+, gsettings-desktop-schemas
+, pkg-config
+, curl
+, openssl
+, jansson
+, wrapGAppsHook
+}:
+
+stdenv.mkDerivation rec {
+  pname = "mediasynclite";
+  version = "0.4.2";
+
+  src = fetchFromGitHub {
+    owner = "iBroadcastMediaServices";
+    repo = "MediaSyncLiteLinux";
+    rev = version;
+    hash = "sha256-mS2e6zZutSEAr5gCS58t0Nzg8dCJgn/yoFMmbgnIM2I=";
+    };
+
+  buildInputs = [
+   gtk3
+   glib
+   gsettings-desktop-schemas
```

Comment:
This probably needs to go to nativeBuildInputs, too.

Reply:
Is there a source that I can find to help me figure out what goes in `nativeBuildInputs` vs `buildInputs`?